Index: pyvisa-py/pyproject.toml
===================================================================
--- pyvisa-py.orig/pyproject.toml
+++ pyvisa-py/pyproject.toml
@@ -61,6 +61,14 @@ changelog = "https://github.com/pyvisa/p
 requires = ["setuptools>=61.2", "wheel", "setuptools_scm[toml]>=3.4.3"]
 build-backend = "setuptools.build_meta"
 
+[tool.setuptools]
+packages=[
+	'pyvisa_py',
+	'pyvisa_py.protocols',
+	'pyvisa_py.testsuite',
+	'pyvisa_py.testsuite.keysight_assisted_tests',
+]
+
 [tool.setuptools_scm]
 write_to = "pyvisa_py/version.py"
 write_to_template = """
