Index: pyvisa-py/pyvisa_py/gpib.py
===================================================================
--- pyvisa-py.orig/pyvisa_py/gpib.py
+++ pyvisa-py/pyvisa_py/gpib.py
@@ -273,6 +273,8 @@ class _GPIBCommon(Session):
         self.interface = None
         if isinstance(self.parsed, GPIBInstr):
             pad = int(self.parsed.primary_address)
+            if self.parsed.secondary_address is not None:
+                sad = int(self.parsed.secondary_address) + 0x60
             # Used to talk to a specific resource
             self.interface = Gpib(
                 name=minor,
